app/routes/changes-enquiries/deferral/functions.js

```diff
@@ -0,0 +1,124 @@
+const i18n = require('i18next');
+const request = require('request-promise');
+
+const dataStore = require('../../../../lib/dataStore');
+const dateHelper = require('../../../../lib/dateHelper');
+const deferralObject = require('../../../../lib/objects/deferralObject');
+const deleteSession = require('../../../../lib/deleteSession');
+const formValidator = require('../../../../lib/formValidator');
+const generalHelper = require('../../../../lib/helpers/general');
+const keyDetailsHelper = require('../../../../lib/keyDetailsHelper');
+const requestHelper = require('../../../../lib/requestHelper');
+
+const root = '/changes-and-enquiries/personal';
+
+const dateRequestReceivedUrl = `${root}/deferral/date-request-received`;
+const defaultDateUrl = `${root}/deferral/default-date`;
+const stopStatePensionUrl = `${root}/stop-state-pension`;
+
+function getDateRequestReceived(req, res) {
+  const awardDetails = dataStore.get(req, 'awardDetails');
+  const details = dataStore.get(req, 'date-request-received', 'deferral');
+  const keyDetails = keyDetailsHelper.formatter(awardDetails);
```

Comment:
Remove keyDetails, same reason as the comment above and for all the other functions

Reply:
done